sbin/pfmon

```diff
@@ -391,7 +411,7 @@ run all runtasks
 sub runtasks {
```

Comment:
I would be better to check if it is the master pfmon process early.
To avoid blocking the child signal.